config/crds/cluster.sigs.k8s.io_machines.yaml

```diff
@@ -711,6 +712,9 @@ spec:
                 up status from the Node to the Machine. It is entirely optional, but
                 useful for end-user UX if it’s present.
               type: string
+          required:
```

Comment:
These are still showing as required.

Reply:
Hmm. I did set them as `+optional` did I miss something else?

Reply:
Maybe it should be a pointer with omitempty? 

/cc @ncdc

Reply:
I don't *think* so, does the crd definition need to be regenerated?

Reply:
Pointer should only be needed if you need to differentiate unset from false.

Reply:
I tried, but there are no changes when I regenerate them

Reply:
Pointer is required for optional fields

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

Reply:
Tried different combinations, and it seems I need `omitempty` to be actually optional

Reply:
so pointer and omitempty it is

Reply:
pushed!